.travis.yml

```diff
@@ -1,13 +1,10 @@
 language: java
-
-script: ./.travis-build.sh
-
 sudo: false
-
-addons:
-  apt:
-    packages:
-      - ant-optional
-
+dist: trusty
 jdk:
   - oraclejdk8
+
+install:
+  - pip install --user  -r ./.travis-requirements.txt
```

Comment:
Should this PR add `.travis-requirements.txt`?

Reply:
If we decide to do CI in a docker container (like what CF does), then this is unnecessary. But if we use a third-party provided OS (like ontology and security-demo using ubuntu from Travis), then `.travis-requirements.txt` should be kept. I am fine with both ways. What do you think?